build/pause/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - build-image-approvers
+reviewers:
+  - build-image-reviewers
```

Comment:
For the record: I still think these are incredible hostile to PR authors vs just inlining a list of users. Even after learning our bespoke yaml format you need to figure out de-ref-ing these by hand.

Reply:
it also obsfucates adding owners. in this case:

https://github.com/kubernetes/kubernetes/blob/6c0278ed45db8a183c0e792a7ce28bbd7b6f9209/OWNERS_ALIASES#L132-L143

cc @dims @cblecker @listx FYI you're being added as reviewers and approvers in a few places

Reply:
@BenTheElder -- do we have a discussion re: OWNERS somewhere?
I don't disagree, but I'm just operating within what exists today.

build-image OWNERS need to be here, because ultimately the changes are being pushed to the staging repos that they're delegated to.

Reply:
I'm just not continuing to approve the status quo without at least protesting that I think it's bad :-)

As for CCing people, I at least would prefer to not become a surprise approver / reviewer, I think at least an FYI comment is in oder :-)

Reply:
> I'm just not continuing to approve the status quo without at least protesting that I think it's bad :-)

Yep. I totally feel that.
For anyone reading that wants to discuss OWNERS, here's what's considered the umbrella issue: https://github.com/kubernetes/community/issues/1808

> As for CCing people, I at least would prefer to not become a surprise approver / reviewer, I think at least an FYI comment is in oder :-)

I caught everyone. Of the five people in `build-image-{approvers,reviewers}`, I'm the PR author, you, Dims, and Christoph were assigned and Linus is in @kubernetes/release-engineering.